crates/isograph_config/src/compilation_options.rs

```diff
@@ -28,6 +28,23 @@ pub struct CompilerConfig {
 #[derive(Default, Debug, Clone, Copy)]
 pub struct ConfigOptions {
     pub on_invalid_id_type: OptionalValidationLevel,
+    pub generate_file_extensions: OptionalGenerateFileExtensions,
+}
+
+#[derive(Default, Debug, Clone, Copy)]
+pub enum OptionalGenerateFileExtensions {
+    Yes,
```

Comment:
Can we make this a boolean please? Otherwise lgtm!! Thank you for tackling this

Reply:
`boolean` in `isograph.config.json` and map it to `enum` in code?